src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorSyntaxNodeExtensions.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.AspNetCore.Razor.Language.Syntax;
+using Microsoft.AspNetCore.Razor.LanguageServer.Formatting;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Extensions;
+
+internal static class RazorSyntaxNodeExtensions
+{
+    public static int GetLeadingWhitespaceLength(this SyntaxNode node, FormattingContext context)
```

Comment:
These are direct copies from the old methods with no modifications. `FormattingContext` is a type specific to this layer, so it can't move.

Reply:
Looks like it's only used for tabsize, maybe we can just take that directly?

Reply:
Actually, it looks like this extension is only needed by one method in one file. I'll just move it to a local function.